src/Test/TestUnitMovement.java

```diff
@@ -161,23 +161,23 @@ private boolean testFuelCosts()
     multiPath.addWaypoint(2, 5);
 
     // Make sure the action didn't actually execute.
-    boolean testPassed = validate(grassPath.getFuelCost(testCo1.getUnitModel(UnitModel.TROOP), testMap) == 7,
+    boolean testPassed = validate(grassPath.getFuelCost(new Unit(testCo1, testCo1.getUnitModel(UnitModel.TROOP)), testMap) == 7,
```

Comment:
Not a huge fan of this either, especially since it was precipitated by a change that also feels unnecessary.

Reply:
Can't argue with that.